packages/calypso-e2e/src/media-helper.ts

```diff
@@ -111,6 +133,6 @@ export function createTestAudio(): string {
  *
  * @returns {string} Full path on disk to the generated test file.
  */
-export function createInvalidFile(): string {
+export function createUnsupportedFile(): TestFile {
```

Comment:
The file it generates isn't `invalid` per se, just that it isn't supported on WPCOM.